com.civicrm.hrjobroles/scss/hrjobroles.scss

```diff
@@ -17,3 +17,10 @@
     position: relative;
   }
 }
+
+
+// workaround to make sure that the texarea always go full-width
+// the !important is necessary to override the #bootstrap-theme style
+.editable-textarea {
+  display: block !important;
```

Comment:
Still I think its better to keep the style inside `.job-roles` namespace with `!important`. I mean thats what the namespace is for, to contain all styles related to `.job-roles`